src/views/Home/Home.jsx

```diff
@@ -165,17 +165,19 @@ const Home = () => {
 			{doneLoading && hasResults ? (
 				<div className="results">
 					<h3>{`${i18n.resultsFor[language]}: ${keyword}`}</h3>
-					<div
-						className={
-							showBestBet && showDefinition
-								? 'results__feature--bestbet--definition'
-								: 'results__feature'
-						}>
-						{showBestBet && (
-							<BestBet language={language} results={stateBestBetResult} />
-						)}
-						{showDefinition && <Definition {...stateDefinitionResult} />}
-					</div>
+					{isFirstPage && (
```

Comment:
Do we need `isFirstPage` since that's included in both `showBestBet` and `showDefinition`.

Reply:
Yes, why not? This way, there is no empty container sitting at the top of the page.  The isFirstPage check can actually be removed from the showBestBet and showDefinition checks in this scenario.

Reply:
👍 Valid reason for leaving this here on the empty container, but might want to leave that condition on showBestBet and showDefinition just in case it's used further down the line and since those flags actually are currently driven by that requirement.